### PR TITLE
V2.2 alpha (now beta)

### PR DESCRIPTION
**Pull request by sasha240100**
### Changelog:
- New examples page look.
- Sass -> less
- webpack 4
- babel 7
- fix tests
- refactor gulp files
- partial examples restructuring

### CODE REVIEW

File: examples/assets/js/script.js [hunk 1]
```diff
@@ -1,40 +1,129 @@
...
-  return check;
-})();
+var iframe = document.querySelector('iframe');
```
Comment (hirako2000) [inline examples/assets/js/script.js:3]: no es6?

File: examples/assets/js/script.js [hunk 2]
```diff
@@ -1,40 +1,129 @@
...
+      item.parentNode.style.display = 'block';
+    } else {
+      item.parentNode.style.display = 'none';
+    }
+  })
+});
 
-  categories.forEach(function (cat) {
-    const isHidden = [].slice.call(cat.querySelectorAll('.minor')).every(item => item.style.display === 'none');
+// SIDEBAR
```
Comment (hirako2000) [inline examples/assets/js/script.js:73]: need commented out code?

**Review by hirako2000 — APPROVED** [review]

File: examples/assets/js/script.js (hunk 1 above)
Comment (sasha240100) [reply examples/assets/js/script.js:3]: @hirako2000 Yes, this file is not processed by webpack. It's included within a script tag.

File: examples/assets/js/script.js (hunk 2 above)
Comment (sasha240100) [reply examples/assets/js/script.js:73]: It's WIP yet. It includes just basic functionality to have examples page working. Let's merge this as a "beta" (not yet stabilized)

**Pull request merged**